roles/internal/grok/tasks/install.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+- name: Install Grok dependencies
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - cmake
+    - libpng16-dev
+    - libtiff-dev
+    - liblcms2-dev
+
+- name: Clone Grok
+  git:
+    repo: https://github.com/GrokImageCompression/grok.git
+    dest: "{{ grok_clone_directory }}"
+    version: "{{ grok_version_tag }}"
+
+- name: Make build directory
+  file:
+    path: "{{ grok_clone_directory }}/build"
+    state: directory
+    mode: 0755
```

Comment:
weird ansible gotcha here, but always put the mode in quotes since its an octal number or it'll be interpreted as decimal, and shit will get weird haha.

https://github.com/ansible/ansible/issues/9196
https://blog.tyk.nu/blog/ansible-and-unix-file-permissions/